### PR TITLE
Update api

### PR DESCRIPTION
**Pull request by ffreitasalves**
The old api_root is no longer working, also some resources had to be updated

### CODE REVIEW

**Comment by daneoshiga** [comment]
Sorry, somehow I didn't get an email for your PR, really sorry, I'm gonna review this asap.

**Comment by ffreitasalves** [comment]
That's OK @daneoshiga . I should have opened an issue.

File: tapioca_jarbas/resource_mapping.py [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
-    },
-    'applicant': {
-        'resource': 'applicant',
-        'docs': 'https://github.com/datasciencebr/jarbas#get-apiapplicant',
-    },
-    'company': {
-        'resource': 'company/{cnpj}',
-        'docs': 'https://github.com/datasciencebr/jarbas#get-apicompanycnpj',
-    },
+  'reimbursement': {
```
Comment (daneoshiga) [inline tapioca_jarbas/resource_mapping.py]: keep 4 spaces of identation here.

**Comment by daneoshiga** [comment]
version 1.0.0 released, thanks for your PR :)

**Pull request merged**